test/packaging/ansible/shutdown-and-terminate-alerts.yml

```diff
@@ -0,0 +1,161 @@
+---
+
+- name: install agent linux
```

Comment:
This is the old test file, we keep it for troubleshooting. Is not run by tests, was replaced by shutdown-and-terminate.yml